influx-writer/influx-writer.py

```diff
@@ -117,16 +118,19 @@ def write(self, listen_dicts):
 
     def start(self):
         self.log.info("InfluxWriterSubscriber started")
+        print("InfluxWriterSubscriber started")
 
         self.register(SUBSCRIBER_NAME)
         while True:
             try:
                 count = self.subscriber()
             except NoSubscriberNameSetException as e:
                 self.log.error("InfluxWriterSubscriber has no subscriber name set.")
+                print("InfluxWriterSubscriber has no subscriber name set. Exiting.")
                 return
             except WriteFailException as e:
                 self.log.error("InfluxWriterSubscriber failed to write: %s" % str(e))
+                print("InfluxWriterSubscriber failed to write, skipping this batch of data.")
```

Comment:
That's fine for now. Is autobuild enabled for this project on docker hub ? If yes, merge and test the new container in prod, so we can sort out the issue.